project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/.github/workflows/docgen_from_Jira.yaml

```diff
@@ -37,9 +37,11 @@ jobs:
  
       - name: Create commit
         run: |
-          git config user.name github-actions
-          git config user.email github-actions@github.com
-          git commit -m "Update from JIRA." {{ cookiecutter.series }}-{{ cookiecutter.serial_number }}.tex 
+          git config user.name $GITHUB_ACTOR
+          git config user.email $GITHUB_ACTOR@users.noreply.github.com
+          git add  --ignore-missing attachments jira_imgs
```

Comment:
Again, I think you want to include the base tex file here? Or does that not get modified? Other than that, this looks good-to-go.